airgun/views/host.py

```diff
@@ -206,7 +206,7 @@ class HostsView(BaseLoggedInView, SearchableViewMixin):
         './/table',
         column_widgets={
             0: Checkbox(locator=".//input[@class='host_select_boxes']"),
-            'Name': Text("./a"),
+            'Name': Text('//*[@id="content"]/table/tbody/tr/td[5]/a'),
```

Comment:
Can we try a better locator?Something like 
'Name': Text("//a[contains(@href, '/new/hosts')]")